src/main/scala/com/lightbend/coursegentools/Helpers.scala

```diff
@@ -238,6 +238,12 @@ object Helpers {
       .runAndExitIfFailed(toConsoleRed(s"Failed to initialize linearized git repository in ${linearizedProject.getPath}"))
   }
 
+  def addGitignoreFromMaster(mainRepo: File, linearizedProject: File): Unit = {
```

Comment:
Can we rename the method to `addGitignoreFromMain` to make it consistent with others, considering methods for eg: `removeExercisesFromCleanMain`
```suggestion
  def addGitignoreFromMain(mainRepo: File, linearizedProject: File): Unit = {
```